app/Http/Requests/UserRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UserRequest extends FormRequest
+{
+    public function authorize(): bool
+    {
+        return false;
+    }
+
+    public function rules(): array
+    {
+        return [
+            'name' => 'required',
+            'email' => 'required|email',
+            'password' => [
+                'required',
+                'regex:/^((?=.*[0-9])(?=.*[a-z])(?=.*([A-Z]|[!-\/:-@[-`{-~]))|(?=.*[A-Z])(?=.*[!-\/:-@[-`{-~])(?=.*([0-9]|[a-z])))[a-zA-Z0-9!-\/:-@[-`{-~]{8,}$/u',
```

Comment:
https://laravel.com/docs/11.x/validation#validating-passwords

Reply:
But why not regex? Besides, is it possible to validate for only 3 of 4 rules with this? I think the regex is available here for a purpose.

Reply:
If laravel options are not enough then ok, but usually it's better to use things that are more understandable at first sight